src/lib/builders/accordion/create.ts

```diff
@@ -86,7 +87,7 @@ export const createAccordion = <Multiple extends boolean = false>(
 
 				return {
 					'data-state': isSelected(itemValue, $value) ? 'open' : 'closed',
-					'data-disabled': disabled ? true : undefined,
+					'data-disabled': disabledAttr(!!disabled),
```

Comment:
I don't think we should required the `!!`. If its `undefined` we treat it as false inside the function.

Reply:
It is currently required for type-safety since `disabledAttr` takes in a `boolean`:

https://github.com/melt-ui/melt-ui/blob/2201f0adc75e20601b0748710110e4a81c56d441/src/lib/internal/helpers/attr.ts#L1-L3

Would it make more sense to take in `boolean | undefined`?

Reply:
> Would it make more sense to take in boolean | undefined?

Yes!